### PR TITLE
 cleanup of files for MrDoob's code style

### PR DESCRIPTION
**Pull request by gero3**
This is a codestyle improvement PR for the src folder and the examples/js folder.

### CODE REVIEW

**Comment by WestLangley** [comment]
This doesn't seem to be working in `examples/js/CurveExtras.js`, for example. Although, using http://zz85.github.io/mrdoobapproves/, that file reformats nicely.

It is largely just adding blank lines to all files, and not reformatting the code itself.

**Comment by mrdoob** [comment]
I think we should only run this in the `src` folder. There are libraries in `examples` that we shouldn't touch.

**Comment by gero3** [comment]
Except the ones in 'examples/js/libs', any other file in examples/js should comply with MDCS, I think.
Or are there other exceptions  to this rule??

**Comment by mrdoob** [comment]
```
examples/js/loaders/ctm
examples/js/loaders/gltf
examples/js/loaders/sea3d
examples/models
examples/obj
examples/scenes
```

**Comment by gero3** [comment]
I updated the branch to exclude the nessecary files.

**Comment by WestLangley** [comment]
Thanks, @gero3!

**Comment by gero3** [comment]
Is there anything else I need to do to get this one or a similar one merged in??

**Comment by WestLangley** [comment]
Yes. Please merge -- or redo and merge.

**Comment by mrdoob** [comment]
Do you mind doing it again? There seem to be conflicts...

**Comment by gero3** [comment]
I fixed the conflicts now.

**Comment by mrdoob** [comment]
I think I wouldn't touch `examples/js/ImprovedNoise.js` nor `examples/js/SimplexNoise.js`. Maybe we could move them to `js/libs`.

**Comment by mrdoob** [comment]
`examples/js/loaders/UTF8Loader.js` is also developed outside this repo...

**Comment by gero3** [comment]
I added a new rule to the MDCS standard dictating that a curlybraces shouldn't be on a new line. this fixes the formatting. 

I also reverted those 3 files.

**Comment by gero3** [comment]
I added 2 rules indicating that else isn't allowed to be on a newline and that there is a space behind a comma.

File: examples/js/ShaderSkin.js [hunk 1]
```diff
@@ -338,26 +338,26 @@ THREE.ShaderSkin = {
...
-			"tDiffuse"	: { type: "t", value: null },
-			"tNormal"	: { type: "t", value: null },
+				"tDiffuse"	: { type: "t", value: null },
+				"tNormal"	: { type: "t", value: null },
 
-			"tBlur1"	: { type: "t", value: null },
-			"tBlur2"	: { type: "t", value: null },
-			"tBlur3"	: { type: "t", value: null },
-			"tBlur4"	: { type: "t", value: null },
+				"tBlur1"	: { type: "t", value: null },
```
Comment (gero3) [inline examples/js/ShaderSkin.js]: Is there a reason why there is so much space here??

Comment (WestLangley) [inline examples/js/ShaderSkin.js]: They are tabs. The larger code block used to line up.

Do we have a rule that omits tabs after the 1st non-tab character?

Comment (WestLangley) [inline examples/js/ShaderSkin.js]: Or should we? : - )

Comment (gero3) [inline examples/js/ShaderSkin.js]: The problem with this file is that even with these tabs is doens't outline correctly.

https://github.com/mrdoob/three.js/pull/6863/files#diff-a1806af1528eb3e8af7952cfb105a597R46
vs
https://github.com/mrdoob/three.js/pull/6863/files#diff-a1806af1528eb3e8af7952cfb105a597R43

the colon doesn't matchup compared to each other .

**Comment by mrdoob** [comment]
Lets do this! 😁

**Comment by mrdoob** [comment]
Thanks @gero3!

**Pull request merged**